ssqatest/tests/frontend/latha_frontend_tests/test_register_new_user.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
Few comments on this one:
* Why are you automation the same test again? The test already exists (I think it existed on the base): https://github.com/atha123/demostore_frontend_backend_tests/blob/main/ssqatest/tests/frontend/my_account/test_register_new_user.py

* Why are you creating tests in this folder 'latha_frontend_tests'? why did you need a new folder. Why not just do it in 'frontend' folder. Remember this whole framework is yours so when recuriter/manger looks at it they're is not reason to show 'latha' vs 'other'. Eventually there will be 120 tests and 100 of them are what you wrote so technically this is all you. I do not agree in creating a different folder. But not a  blocker. Just my opinion.